src/mixins/style-propable.js

```diff
@@ -33,7 +33,12 @@ export default {
   // only call it when passing style attribute to html elements.
   // If you call it twice you'll get a warning anyway.
   prepareStyles() {
```

Comment:
Actually, *yes*, you can! That's a huge help. Babel compiles:

```js
function foo (...args) {
  return args;
}
```

to:

```js
function foo() {
  for (var _len = arguments.length, args = Array(_len), _key = 0; _key < _len; _key++) {
    args[_key] = arguments[_key];
  }

  return args;
}
```

which will not deopt.

Nice suggestion!